pkgs/development/python-modules/beancount/default.nix

```diff
@@ -51,6 +52,12 @@ buildPythonPackage rec {
     mv beancount tests
   '';
 
+  disabledTestPaths = lib.optionals stdenv.hostPlatform.isDarwin [
+    # Cannot run the gpg-agent. If needed, implement as passthru tests.
+    "tests/utils/encryption_test.py::TestEncryptedFiles::test_read_encrypted_file"
+    "tests/utils/encryption_test.py::TestLoadIncludesEncrypted::test_include_encrypted"
```

Comment:
Is this pattern (adding `file::Class::test_name` to `disabledTestPaths`) encouraged? documented? discouraged?

The more consensual thing would be to add the test names to `disabledTests`. I am not sure what is the SOTA for this in nixpkgs.

Reply:
Sounds good. I just confirmed these test names only happen in one place. Done.